scripts/README.md

```diff
@@ -37,13 +37,21 @@ In case of a problem, another method to generate the changelog is available at t
 
 2. Clean the generated changelog, to match the format of [https://github.com/mui/mui-x/releases](https://github.com/mui/mui-x/releases).
 3. Update the root `package.json`'s version
-4. Update the versions of the other `package.json` files and of the dependencies with `pnpm release:version` (`pnpm release:version prerelease` for alpha / beta releases).
-   Do not skip the version bump if Lerna detects a change in the package. It is important to release the package if there are **any** changes to it.
-   If Lerna doesn't suggest a version bump for the package, don't release it.
+4. Update the versions of the other `package.json` files and of the dependencies with `pnpm release:version` (`pnpm release:version prerelease` for alpha / beta releases). [Check the warning below]
 5. Open PR with changes and wait for review and green CI.
 6. Once CI is green and you have enough approvals, send a message on the `team-x` slack channel announcing a merge freeze.
 7. Merge PR.
 
+:::warning
+
+Make sure of the following when versioning the packages with `release:version`:
+
+- Do not skip the version bump if Lerna detects a change in the package. It is important to release the package if there are **any** changes to it.
+- If Lerna doesn't suggest a version bump for the package, don't release it.
+- When releasing a package, make sure to sync the version of the package with the version of the root `package.json` file.
+
+:::
+
```

Comment:
```suggestion
4. Update the versions of the other `package.json` files and of the dependencies with `pnpm release:version` (`pnpm release:version prerelease` for alpha / beta releases).
  > [!WARNING]
  > Make sure of the following when versioning the packages with `release:version`:
  >
  > - Do not skip the version bump if Lerna detects a change in the package. It is important to release the package if there are **any** changes to it.
  > - If Lerna doesn't suggest a version bump for the package, don't release it.
  > - When releasing a package, make sure to sync the version of the package with the version of the root `package.json` file.
5. Open PR with changes and wait for review and green CI.
6. Once CI is green and you have enough approvals, send a message on the `team-x` slack channel announcing a merge freeze.
7. Merge PR.

```

For better formatting on GitHub:
<img width="1028" src="https://github.com/user-attachments/assets/838b59a3-e361-488c-a49a-59f971aa5adb" />